rpc/types/events.go

```diff
@@ -0,0 +1,210 @@
+package types
+
+import (
+	"encoding/json"
+	"strconv"
+
+	"github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+	evmtypes "github.com/tharsis/ethermint/x/evm/types"
+)
+
+// EventFormat is the format version of the events
+type EventFormat int
+
+const (
+	eventFormatUnknown EventFormat = iota
+	eventFormat1
+	eventFormat2
```

Comment:
What are these formatN? can you leave a comment on what they represent?

Reply:
I moved some comments from the `ParseTxResult` function to here, better visibility.